### PR TITLE
chore(bookmarks): separate files by pageId

### PR DESCRIPTION
**Pull request by dryabinin-actionengine**

### CODE REVIEW

File: src/components/layers-panel/layers-panel.tsx [hunk 1]
```diff
@@ -170,6 +172,21 @@ export const LayersPanel = ({
   onPointToLayer,
 }: LayersPanelProps) => {
   const layout = useAppLayout();
+  const location = useLocation();
+
+  const bookmarksPageId = useMemo(() => {
+    switch (location.pathname) {
```
Comment (belom88) [inline src/components/layers-panel/layers-panel.tsx:178]: I would send the id from the page via props

File: src/utils/bookmarks-utils.ts [hunk 2]
```diff
@@ -112,3 +114,13 @@ const convertArcGisCameraPositionToBookmarkViewState = (camera: any): LayerViewS
 
   return null;
 }
+
+export const checkBookmarksByPageId = (bookmarks: Bookmark[], pageId: BookmarkPageId): BookmarkPageId => {
```
Comment (belom88) [inline src/utils/bookmarks-utils.ts:118]: JSDoc requred

**Review by belom88 — CHANGES_REQUESTED** [review]

File: src/components/layers-panel/layers-panel.tsx (hunk 1 above)
Comment (dryabinin-actionengine) [reply src/components/layers-panel/layers-panel.tsx]: Done

File: src/utils/bookmarks-utils.ts (hunk 2 above)
Comment (dryabinin-actionengine) [reply src/utils/bookmarks-utils.ts]: Added

**Review by belom88 — APPROVED** [review]

**Pull request merged**